src/pages/my-patches/patch-card/BuildStatusIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { BuildStatus } from "types/build";
+import { RunningIcon } from "pages/my-patches/patch-card/BuildStatusIcon/RunningIcon";
+import { SucceededIcon } from "pages/my-patches/patch-card/BuildStatusIcon/SucceededIcon";
+import { FailedIcon } from "pages/my-patches/patch-card/BuildStatusIcon/FailedIcon";
+import { CreatedIcon } from "pages/my-patches/patch-card/BuildStatusIcon/CreatedIcon";
+import Tooltip from "@leafygreen-ui/tooltip";
+
+interface Props {
+  status: string;
+  buildVariant: string;
+}
+
+export const BuildStatusIcon: React.FC<Props> = ({ status, buildVariant }) => {
+  const icon = statusToIcon[status];
+  if (!icon) {
+    return null;
+  }
+
+  return (
+    <Tooltip
+      triggerEvent="hover"
+      trigger={<div>{icon}</div>}
```

Comment:
Is this div necessary. Can we just place the icon in instead?

Reply:
It is necessary for the tooltips to show up on the svg icons.